test/scripts/tags/post_link.js

```diff
@@ -57,11 +57,11 @@ describe('post_link', () => {
       .should.eql('<a href="/title-with-tag/" title="This is a &lt;b&gt;Bold&lt;&#x2F;b&gt; &quot;statement&quot;">This is a <b>Bold</b> "statement"</a>');
   });
 
-  it('no slug', () => {
-    postLink([]).should.eql('<a href="#">Post not found: Invalid post_link</a>');
+  it('should throw if no slug', () => {
+    should.throw(() => postLink([]), Error, /Post not found: no slug for post_link\./);
   });
 
-  it('post not found', () => {
-    postLink(['bar']).should.eql('<a href="#">Post not found: bar</a>');
+  it('should throw if post not found', () => {
+    should.throw(() => postLink(['bar']), Error, /Post not found: post_link bar\./);
```

Comment:
And the unit tests, too.

Reply:
I am not sure what this diff means, but I updated the test to match the requested changes (4fdac0db5e359c0f57fa4f12c5720f72c000731f).